File: comiclysm-app/src/app/models/Trade.ts
export class Trade {

    tradeId: number;
    tradeComicToTradeId: number;

    constructor() {}


    setTradeComicToTradeId( comicToTradeId: number) {
        this.tradeComicToTradeId = comicToTradeId;
    }

    getTradeId() {
        return this.tradeId;
    }
    getTradeComicToTradeId() {
        return this.tradeComicToTradeId;
    }
}

File: comiclysm/src/main/java/com/comiclysm/project/comiclysm/models/Trade.java
package com.comiclysm.project.comiclysm.models;

import javax.persistence.*;

@Entity
@Table(name = "trade")
public class Trade {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "trade_id")
    int tradeId;


    @Column(name = "trade_comic_to_trade_id")
    int tradeComicToTradeId;

    public Trade() {}

    public Trade(int tradeComicToTradeId) {
        this.tradeComicToTradeId = tradeComicToTradeId;
    }

    public int getTradeId() {
        return tradeId;
    }

    public void setTradeId(int tradeId) {
        this.tradeId = tradeId;
    }

    public int getTradeComicToTradeId() {
        return tradeComicToTradeId;
    }

    public void setTradeComicToTradeId(int tradeComicToTradeId) {
        this.tradeComicToTradeId = tradeComicToTradeId;
    }
}
